### DISCUSSION TITLE
How to make beets see files with accent marks?

### DISCUSSION THREAD

**Discussion by phirestalker** [beetbox/beets#4519] [Q&A]
These files were imported (and moved) by beets, so I can assume that it understood the filenames then. I may have changed my locale setting sometime after import. Now, when I use the -p option to list the path to the file the system does not recognize it, but when I list the files in that directory the name looks the same to me. I am unsure what to do about this.

**Comment by sampsyo** [comment]
Unfortunately, this kind of encoding issue can be *extremely* subtle and hard to track down. If your locale settings did in fact change, a good option may be to import the relevant files again. That is, you could try moving your database file aside (as a backup) and recreating it with `beet import -A`.

**Reply by phirestalker** [reply]
What if I keep the database file (still backing it up) and perform an import on each album directory? Beets would see that it is in the database already and when it thinks there are duplicates I could choose overwrite. Would doing this make more of a mess?

The reason I ask, is it would be easier to keep track of which ones were correctly imported and which ones are really missing.

**Reply by sampsyo** [reply]
Yes, beets would think there were duplicates in your library. You could pick "remove old" at the duplicate prompt, but I'm concerned that things could go very wrong because _some_ files may be correctly tracked and others not… so I'd be very careful here! Certainly making a backup of the whole music library would be important.

**Reply by phirestalker** [reply]
I think I understand what might happen. So the safer way might be to copy the directory elsewhere and then reimport them from the copy. That way if it needs to replace "duplicates" it will still have all the songs.

**Reply by sampsyo** [reply]
Yep, that's a good idea.
